test/tmp_file.go

```diff
@@ -29,7 +30,8 @@ var appFs = afero.NewOsFs()
 func TmpDir(t TestReporter, dir string) string {
 	fullPath := dir
 	if !path.IsAbs(dir) {
-		fullPath = fmt.Sprintf("%s/%s/%s", os.TempDir(), randomAlphaNumeric(), dir)
+		tmpDir := filepath.Clean(os.TempDir())
```

Comment:
Fixes https://github.com/golang/go/issues/21318 - need a comment?

Reply:
A quick comment would be helpful. Else it's always in history but...

Reply:
e2c3fbd